### PR TITLE
Update tvr.json

### PR DESCRIPTION
**Pull request by Camtendo**
You guys should propose some facts too.

Closes https://github.com/The-Velvet-Room/web-overlay/issues/140

### CODE REVIEW

**Comment by dguenther** [comment]
"Camtendo is also a professional FFXIV and Dragon Nest player"

**Comment by nswartz** [comment]
ayyyyyyyyyyyyyy

**Comment by nswartz** [comment]
"We're not sure who Dr. Dank is."

**Comment by dguenther** [comment]
So these facts can be about anyone in the local scene? Maybe something about David playing Guilty Gear?

**Comment by Camtendo** [comment]
Yeah, anything. I'll add some more later. Let's not merge this until the Friday before the Arcadian to give us time to think.

**Comment by Camtendo** [comment]
From Dr. Dank

`I can finish the Rubik's cube in 2 minutes`

File: public/resources/tvr.json [hunk 1]
```diff
@@ -20,4 +20,11 @@
...
 "My name is Elizabeth. I will teach you wonderful facts while you wait for more action.",
-"JBM gained his nickname as The Gatekeeper by beating Darkrain at Roll Tier X4: Remix."]
\ No newline at end of file
+"JBM gained his nickname as The Gatekeeper by beating Darkrain at Roll Tier X4: Remix.",
+"Cam2 got his tag by being the second Cam in the NE Smash scene. I think we should start calling him Cam2King.",
+"GQDang was our first Iowa guest on Pardon the Smash. Word has it that he can play Megaman really fast. Could you beat him?",
+"Unfortunately, Blank did not get his name from No Game No Life.",
+"When Persona 5 gets released, will I be replaced? I sure hope not.",
+"My favorite fighting game is Persona 4 Arena. There's a similar game called Guilty Gear. Davidcue is supposedly the best in the United States at it. Exciting!",
+"Dr. Dank name isn't just for show. I heard he can finish a Rubik's Cube within 2 minutes. That's quite impressive.",
```
Comment (dguenther) [inline public/resources/tvr.json]: Should be `Dank's`

**Pull request merged**